pivot/test_resources/csvfiles/Student_Data_All_Majors.csv

```diff
@@ -1 +0,0 @@
-major_abbr,pathway,College,iqr_min,q1,median,q3,iqr_max
```

Comment:
We don't need this anymore because the actual `Student_Data_All_Majors.csv` file containing the data of the aggregated years will be `Student_Data_All_Majors_All_Years.csv`